core/src/main/java/org/elasticsearch/action/admin/indices/validate/query/TransportValidateQueryAction.java

```diff
@@ -197,7 +197,8 @@ protected ShardValidateQueryResponse shardOperation(ShardValidateQueryRequest re
             valid = false;
             error = e.getMessage();
         } finally {
-            SearchContext.current().close();
+            searchContext.clearReleasables(SearchContext.Lifetime.COLLECTION);
+            searchContext.close();
```

Comment:
close is supposed to clear as well so this shouldn't be necessary to call clearReleasables